Frontend/ui-driver/src/MerchantConfigs/DefaultConfig.purs

```diff
@@ -74,6 +74,10 @@ config =
   , profile : { 
     bookingOptionMenuForTaxi : false
   }
+  , waitTimeConfig : {
+    enableWaitTime : true,
```

Comment:
wait time has to be enabled for all merchants? I think so far we are not having waitTime in YS enabled

Reply:
@witcher-shailesh  I can't see in code that we have disabled it for YS, It's enabled for all